src/contents/all/components/app/App.tsx

```diff
@@ -171,6 +171,10 @@ const App = () => {
     }
   }
 
+  const imageClick = () => {
+    console.log('Click');
+  }
+
```

Comment:
Let's just remove this and we're good to go!

Reply:
whoops!